src/commitlint/cli.py

```diff
@@ -228,6 +228,10 @@ def main() -> None:
         console.error(f"{ex}")
         sys.exit(1)
 
+    except FileNotFoundError as ex:
+        console.error(f"{ex}")
```

Comment:
Please provide a user-friendly error. The current output is something like `"[Errno 2] No such file or directory: 'path/to/file'"`.